pyproject.toml

```diff
@@ -71,12 +72,9 @@ distributed = ['distributed', 'dask[distributed]']
 s3 = ['boto3', 'botocore']
 netcdf = ['netcdf4']
 postgres = ['psycopg2']
-loader = ['odc-loader']
```

Comment:
There are still imports from `odc.loader` in the code, it's just now the dependency is no longer optional because of `odc-stac`. Need to add `odc-loader` to `dependencies` list, even though it will be always pulled in by `odc-stac` also.